nevergrad/functions/olympus/core.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Based on https://github.com/aspuru-guzik-group/olympus
+
+import matplotlib.pyplot as plt
+import numpy as np
+from nevergrad.parametrization import parameter as p
+from ..base import ExperimentFunction
+import olympus
+
+
+class Olympus(ExperimentFunction):
+    def __init__(self, kind : str, dimension: int = 10) -> None:
+        self.kind = kind
+        self.param_dim = dimension
+        super().__init__(self._simulate_traditional_surface, p.Array(shape=(dimension,)))
+    
+    def _simulate_traditional_surface(self, x: np.ndarray) -> float:
+        assert self.kind in ["AckleyPath", "Dejong", "HyperEllipsoid",
+                        "Levy", "Michalewicz", "Rastrigin", "Rosenbrock", 
+                        "Schwefel", "StyblinskiTang", "Zakharov"]
+
```

Comment:
Sounds correct to me, but for making assessment faster we can average noise by specifying that for a posteriori evaluation we might average 42 runs as follows (this does not impact the run, just the evaluation of the final solution at the end...):

def evaluation_function(self, *recommendations) -> float:
        """Averages multiple evaluations if necessary."""
        x = recommendations[0].value
        losses = [surface.run(x)[0][0] for _ in range(42)]
        return sum(losses) / len(losses)

Reply:
Even better (if possible) would be to have "def evaluation_function" return the value without noise (while still using the noisy one in super.__init__(....., ...) ).